kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -1227,7 +1280,9 @@ public TaskInstance findTaskById(Long taskId) {
             ServiceResponse<TaskSummaryList> response = (ServiceResponse<TaskSummaryList>) executeJmsCommand( script, DescriptorCommand.class.getName(), "BPM" ).getResponses().get(0);
 
             throwExceptionOnFailure(response);
-
+            if (shouldReturnWithNullResponse(response)) {
+                return null;
+            }
```

Comment:
I guess we can't much with this right now, but it seems to me that needing to add the same check to dozens of places is far from ideal.